### PR TITLE
Feat/223 add narrative video

### LINKED ISSUE #225: For the videos embeded in docs, when selecting model org -> when you go to doc store first time click file see video, after that, just card with small video icon

### PR DESCRIPTION
**Pull request by katjam**
Fixes #225, Partial #223

## Description

- Add videos to documents with first time & subsequent state
- Improve sizing of videos everywhere & share markup
- Replace intro video

@TheLabCollective/developers

### CODE REVIEW

**Comment by katjam** [comment]
@Nikomus Please improve visual & layout of the videos in documents.In particular, the Watch video button has temp icon (eye) &  not the right size / placement.

Also create an issue for any accessibility improvements you see opportunity for. See #223 for ref of further intended work.

File: src/styles/app.scss [hunk 1]
```diff
@@ -25,6 +25,10 @@ body {
   background-color: var(--black);
 }
 
+iframe {
```
Comment (katjam) [inline src/styles/app.scss:28]: Oh probably don't need this anymore. Should I remove?

File: src/styles/video.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.video-container {
+  height: 0px;
+  padding-top: 56.25%;
```
Comment (Nikomus) [inline src/styles/video.scss:3]: I think you can just use aspect-ratio for this.

File: src/styles/app.scss (hunk 1 above)
Comment (Nikomus) [reply src/styles/app.scss:28]: Width looks like it's already 100%

**Comment by katjam** [comment]
@Nikomus I'm going to set this aside until we speak later.

**Review by Nikomus — APPROVED** [review]
Let's merge this in! 🎥

**Pull request merged**